base/src/main/scala/typeclass/LiskovFunctions.scala

```diff
@@ -0,0 +1,13 @@
+package scalaz
+package typeclass
+
+import Liskov.<~<
+
+trait LiskovFunctions {
+  /** http://typelevel.org/blog/2014/03/09/liskov_lifting.html **/
+  def liftCvf[A, B, F[_]: Functor](a: A <~< B): F[A] <~< F[B] = a.asInstanceOf[F[A] <~< F[B]]
```

Comment:
Also, you can use this cast to get stuck in Scalazzi-safe code, unless you seal Liskov.